src/main/java/seedu/dictionote/model/ModelManager.java

```diff
@@ -25,27 +26,33 @@
     private final FilteredList<Note> filteredNote;
     private final NoteBook noteBook;
     private final FilteredList<Contact> filteredContacts;
+    private final Dictionary dictionary;
+    private final FilteredList<Content> filteredContent;
 
     /**
      * Initializes a ModelManager with the given addressBook and userPrefs.
      */
-    public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs, ReadOnlyNoteBook noteBook) {
+    public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs,
+                        ReadOnlyNoteBook noteBook, ReadOnlyDictionary dictionary) {
         super();
         requireAllNonNull(addressBook, userPrefs, noteBook);
 
         logger.fine("Initializing with dictionote book: " + addressBook
                 + " and user prefs " + userPrefs
-                + "and note book" + noteBook);
+                + "and note book" + noteBook
+                + "and dictionary content" + dictionary);
```

Comment:
A minor issue, but shouldn't there be a space before and after `and note book` and `and dictionary content`?